trio_model.cc

```diff
@@ -172,7 +173,7 @@ RowVector256d TrioModel::PopulationPriors() {
  */
 Matrix16_16d TrioModel::PopulationPriorsExpanded() {
   Matrix16_16d population_priors = Matrix16_16d::Zero();
-  const Matrix16_16_4d kTwoParentCounts = TwoParentCounts();
+  const Matrix16_16_4d kTwoParentCounts = TwoParentCounts();//NOTE: Global const? don't need to recalculate
```

Comment:
This was originally a global const but I removed it because I was not sure if it was appropriate to define it as such in utility.h. If you look, I have a line for it commented out. IIRC every time a file was linking in utility.h, it would call TwoParentCounts() several times (based on valgrind)?